doc/topics/reactor/index.rst

```diff
@@ -98,6 +97,90 @@ can be called:
 This example will execute the state.orchestrate runner and initiate an
 orchestrate execution.
 
+The Goal of Writing Reactor SLS Files
+=====================================
+
+Reactor SLS files share the familiar syntax from Salt States but there are
+important differences. The goal of a Reactor file is to process a Salt event as
+quickly as possible and then to optionally start a **new** process in response.
+
+They must be quick because Reactor sls files are matched and rendered
+sequentially in a single process. Long-running Reactor files will cause other
```

Comment:
This is sort of misleading. The _rendering_ must be quick, but the execution doesn't have to be. (You do point that out down below, but this threw me off.)